julia-mode.el

```diff
@@ -67,6 +67,35 @@ User can still use `abbrev-mode' or `expand-abbrev' to substitute
 unicode for LaTeX even if disabled."
   :type 'boolean)
 
+(defconst julia-mode--latexsubs-partials
+  (let ((table (make-hash-table :test 'equal)))
+    (maphash (lambda (latex subst)
+               (cl-assert (string= (substring latex 0 1) "\\") nil
+                          "LaTeX substitution does not start with \\.")
+               (let ((len (length latex)))
+                 (cl-assert (< 1 len) nil "Trivially short LaTeX subtitution")
+                 (cl-loop for i from 2 to len
```

Comment:
Wouldn't be a bad idea to write a comment here explaining which of `\`, `\a`, `\al` is the first hash put into the table. I find myself having to lookup the docstring of `substring` every single time because I can't keep straight the indexing behavior.